app/scripts/Xprize-UserInterfacesRetargeting.xml

```diff
@@ -51,15 +51,15 @@
     </dependencies>
     <node>icub-virtualizer</node>
   </module>
-  
+
   <module>
     <name>yarp-openvr-trackers</name>
     <environment>YARP_FORWARD_LOG_ENABLE=0</environment>
     <description>Run openvr vive trackers</description>
     <node>icub-virtualizer</node>
     <tag>openvr-trackers</tag>
   </module>
-  
+
```

Comment:
This has to be removed

Reply:
Done in https://github.com/robotology/walking-teleoperation/pull/108/commits/fcb7041e7ff206fbfb0d8c807b1c673d1eae76f3